apps/coscrad-frontend/src/utils/generic-components/presenters/tables/generic-index-table-presenter/__snapshots__/index-table.spec.tsx.snap

```diff
@@ -142,60 +142,64 @@ exports[` 1`] = `
         class="records-table"
       >
         <div>
-          <table>
```

Comment:
We might want a Cypress component test of this table instead. From what I can see, you wrapped in a tiny bit of styling which completely through off the diff. As long as the index tables are rendering properly and the existing Cypress index-to-detail flow tests are good, it should be safe.